src/renderer/families/polkadot/operationDetails.js

```diff
@@ -0,0 +1,364 @@
+/* eslint-disable consistent-return */
+// @flow
+import startCase from "lodash/startCase";
+import React, { useMemo } from "react";
+
+import { BigNumber } from "bignumber.js";
+
+import type { Currency, Unit, Operation, Account } from "@ledgerhq/live-common/lib/types";
+
+import { getDefaultExplorerView, getAddressExplorer } from "@ledgerhq/live-common/lib/explorers";
+
+import { openURL } from "~/renderer/linking";
+
+import {
+  OpDetailsTitle,
+  Address,
+  OpDetailsData,
+  OpDetailsVoteData,
+} from "~/renderer/modals/OperationDetails/styledComponents";
+import { Trans } from "react-i18next";
+import Box from "~/renderer/components/Box/Box";
+import { usePolkadotPreloadData } from "@ledgerhq/live-common/lib/families/polkadot/react";
+import Text from "~/renderer/components/Text";
+import FormattedVal from "~/renderer/components/FormattedVal";
+import CounterValue from "~/renderer/components/CounterValue";
+import { useDiscreetMode } from "~/renderer/components/Discreet";
+
+const helpURL = "https://support.ledger.com/hc/en-us/articles/FIXME";
+
+function getURLFeesInfo(op: Operation): ?string {
+  if (op.fee.gt(200000)) {
+    return helpURL;
+  }
+}
+
+function getURLWhatIsThis(op: Operation): ?string {
+  if (op.type !== "IN" && op.type !== "OUT") {
+    return helpURL;
+  }
+}
+
+function formatPalletMethod(palletMethod: ?string): string {
```

Comment:
This is a formatting util - almost a pure-component but just for text.
This could be put somewhere else, but this one is used only in this file, so IMHO no need to export this as a shared helper.